### ISSUE TITLE
Assets Phalcon\Exception The collection does not exist in the manager

### ISSUE THREAD

**Issue by googlle** [phalcon/cphalcon#11408]
Hello.
I have layout with content
```php
<html>
    <head>
        <title>Site</title>
        <?php $this->assets->outputJs('header') ?>
    </head>
    <body>

        <!-- ... -->

        <?php $this->assets->outputJs('footer') ?>
    </body>
</html>
```
Some controllers don't include javascript
I get this message "Phalcon\Exception The collection does not exist in the manager"

**Comment by googlle** [comment]
I think in the method get() " throw new Exception("The collection does not exist in the manager");" must be replaced by "return new Collection();" like in the method getJs()
```php
/**
* Returns a collection by its id
*
*<code>
* $scripts = $assets->get('js');
*</code>
*/
public function get(string! id) -> <Collection>
{
    var collection;

    if !fetch collection, this->_collections[id] {
        throw new Exception("The collection does not exist in the manager");
    }

    return collection;
}

/**
 * Returns the CSS collection of assets
 */
public function getJs() -> <Collection>
{
    var collection;

    /**
     * Check if the collection does not exist and create an implicit collection
     */
    if !fetch collection, this->_collections["js"] {
        return new Collection();
    }

    return collection;
}
```

**Comment by mahen3d** [comment]
i have the same issue on my web application, but no solution for this for now ? people ask me to try few things but none of them seems to solve the problem

[See question on Stackoverflow](http://stackoverflow.com/questions/36036180/phalcon-asset-management-the-collection-does-not-exist-in-the-manager)

any idea on how to fix the problem ?

**Comment by Jurigag** [comment]
PR with exists method waiting here : https://github.com/phalcon/cphalcon/pull/11558

**Comment by sergeyklay** [comment]
Now you can use `Phalcon\Assets\Manager::exists()`. Implemented in 2.1.x branch

**Comment by googlle** [comment]
Thanks
